src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -98,7 +98,7 @@ export class RundownTimelineService implements RundownService {
     infinitePiecesAfter
       .filter((piece) => !infinitePiecesBefore.includes(piece))
       .forEach((piece) => {
-        const infinitePieceAddedEvent: InfiniteRundownPieceAddedEvent =
+        const infinitePieceAddedEvent: RundownInfinitePieceAddedEvent =
 					this.rundownEventBuilder.buildInfiniteRundownPieceAddedEvent(rundown, piece)
```

Comment:
Am i right in saying that some linting is missing the line being commented on? 
That line looks to be indented way to much. Might only be looking weird on GitHub.

Reply:
I'd also put it on the same line, unless it gets past the max line width, which i think this one might. 
If too long, i'd line break similar to this, but not have the second piece indentet as much as it is here.